spec/models/folio/patron_spec.rb

```diff
@@ -16,6 +16,23 @@
     build(:fee_borrower, custom_properties: {})
   end
 
+  describe '.find' do
+    before do
+      allow(FolioClient).to receive(:new).and_return(folio_client)
+      allow(Honeybadger).to receive(:notify)
+    end
+
+    context 'when response is nil' do
+      let(:folio_client) { instance_double(FolioClient, patron_info: nil) }
+
+      it 'notifies honeybadger' do
+        patron = described_class.find('foo')
+        expect(patron.patron_info).to be_blank
+        expect(Honeybadger).to have_received(:notify)
```

Comment:
Can you add a test that makes sure that when you call patron.patron_info.user_info it doesn't break everything?

Reply:
It doesn't seem to be. I am trying to log in locally and I am getting   [HTTP] < 401 Unauthorized application/json (3.576ms) from patron_info. So the fix isn't working.

Reply:
It looks like self.find isn't being called and patron_info is being passed in from somewhere.

Reply:
Actually might be a different issue, because I am having problems with connecting to folio.